src/main/java/gregtech/common/pipelike/fluidpipe/tile/TileEntityFluidPipeTickable.java

```diff
@@ -122,7 +124,16 @@ private void distributeFluid(int channel, FluidTank tank, FluidStack fluid) {
             IFluidHandler pipeTank = tank;
             CoverBehavior cover = getCoverableImplementation().getCoverAtSide(facing);
             if (cover != null) {
-                pipeTank = cover.getCapability(CapabilityFluidHandler.FLUID_HANDLER_CAPABILITY, pipeTank);
+                IFluidHandler capability = cover.getCapability(CapabilityFluidHandler.FLUID_HANDLER_CAPABILITY, pipeTank);
+                // Shutter covers return null capability when active, special check to prevent NPE
+                if (cover instanceof CoverShutter && capability == null) {
+                    continue;
+                }
+                else if (capability == null) {
+                    GTLog.logger.error("Null capability when attempting to transfer fluids at location  {}", this.getPipePos());
```

Comment:
In case this would happened, this error would flood log as it will be logged every time.